README-KO.md

```diff
@@ -1,6 +1,6 @@
 # Databricks Scala Guide
 
-1000명이 넘는 contributor를 가진 Spark는 빅데이터 분야에서 가장 큰 오픈소스 프로젝트이며, Scala를 사용하는 프로젝트 중 가장 활성화된 프로젝트 입니다. 이 가이드라인은 [Databricks](http://databricks.com/) 엔지니어 팀 뿐 아니라, Spark에 기여하는 엔지니어들의 경험을 바탕으로 만들어 졌습니다.
+Databricks의 엔지니어들은 내부 리포지토리 "universe" 뿐만 아니라, 세계에서 가장 활발하게 개발되고있는 [Apache Spark](https://spark.apache.org), [Delta Lake](https://delta.io/) 와 같은 다양한 Scala기반의 오픈소스 프로젝트들에 기여하고 있습니다. 이 가이드라인은 엔지니어링 팀 및 광범위한 오픈 소스 커뮤니티의 경험을 바탕으로 작성되었습니다.
```

Comment:
> At Databricks, our engineers work on some of the most actively developed Scala codebases in the world, including our own internal repo called "universe" as well as the various open source projects we contribute to, e.g. Apache Spark and Delta Lake. This guide draws from our experience coaching and working with our engineering teams as well as the broader open source community.

I thought we wouldn't mention the private repo but we did in english content. Looks OK to me.